packages/flutter/lib/src/services/browser_context_menu.dart

```diff
@@ -0,0 +1,78 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+
+import 'system_channels.dart';
+
+/// Controls the browser's context menu on the web platform.
+///
+/// The context menu is the menu that appears on right clicking or selecting
+/// text in the browser, for example.
+///
+/// On web, by default, the browser's context menu is enabled by default, and
+/// Flutter's context menus are hidden.
+///
+/// On all non-web platforms, this does nothing.
+class BrowserContextMenu {
+  BrowserContextMenu._();
+
+  /// Ensure that a [BrowserContextMenu] instance has been set up so that the platform
+  /// can handle messages on the scribble method channel.
+  static void ensureInitialized() {
```

Comment:
Why does this have to be eagerly initialized? The engine side doesn't seem to send data to the framework?

Reply:
Also the comment contains "scribble method channel".

Reply:
Whoops, copied and forgot about it.  Thanks.